danish/velkommen.md

```diff
@@ -14,4 +14,6 @@ Vælger du at:
 
 [Tage kapsækken på ryggen og vandre sydpå, "Come so med jer! What are you venting for?"](haervejen/haervejen.md)
 
-[Gå en tur i din have] (have/have.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.